frontend/src/metabase/visualizations/components/ClickActions/ClickActionsPopover.tsx

```diff
@@ -56,6 +56,20 @@ export class ClickActionsPopover extends Component<
     }
   }
 
+  componentDidMount(): void {
+    window.addEventListener("keydown", this.handleKeyDown, { capture: true });
```

Comment:
There already is a keydown listener somewhere because Escape key will sometimes close the popover. Let's figure out why doesn't it always kick in.

In `master`:
1. Browse Products
2. Click "Category" table header
3. Click "Filter by this column"
4. Click "Is" to open up a dropdown
5. Hit Escape

The popover gets closed!
And actually the above behavior is incorrect.
Escape should close the operator picker first, and only a subsequent Escape press should close the popover in question.

----

We should figure out the line that says  `// TODO - come back to this` in `ClickActionsPopover`.

Reply:
How about we just enable `trapFocus` for the popover? It probably makes sense anyway and solves the issue.